schema/xsd/direct.xsd

```diff
@@ -59,6 +59,7 @@
                 </xs:element>
             </xs:choice>
             <xs:element name="signature-type" type="signature-type" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="on-behalf-of" type="signing-on-behalf-of" minOccurs="0" maxOccurs="1"/>
```

Comment:
Det har jeg ikke tenkt på. Det blir jo en tydeligere kontrakt. 

En ulempe er at vi får en statisk default-verdi for direkteoppdrag, og en "dynamisk defaultverdi" i portaloppdrag (som settes i backend), men det er vel helt greit, i grunn.